docker/docker-compose.yml

```diff
@@ -28,3 +28,27 @@ services:
       - 80:80
     volumes:
       - ./mediator.conf.template:/etc/nginx/templates/default.conf.template:z
+  redis:
+    image: "redis:alpine"
+
+    ports:
+     - "6379:6379"
+
+    volumes:
+      - ./redis-data:/var/lib/redis
+      - ./redis.conf:/etc/redis/redis.conf
+
```

Comment:
does Redis need a health check? should the agent depend on Redis?

Reply:
The health check is something Daniel recommended be added in, I think it's not a bad idea to have one

Reply:
Redis should have a healthcheck and then the agent should have a depends on condition of `service_healthy` for the service.